tetris build fix and add esc to quit tetris

danamlund opened the pull request:
Besides the Makefile SRC, I could not find other reasons why tetris would not work on this keyboard/keymap.

danamlund commented:
I just realized that I forgot to call the tetris_start(..) function. Its amazing how you can mess up when you cannot test software.

This pull request should work a lot better now

Maartenwut commented:
It works, which is very nice, but for some reason it makes two 'playing fields' instead of one.
https://youtu.be/Bx1wEQelE70

danamlund commented:
I updated the merge with the other home/end and other changes I also made to the official pull request.

I think I fixed the drawing-board-twice bug. I now only draw the board on a key down event.
